src/app/qgisapp.h

```diff
@@ -2072,6 +2073,8 @@ class APP_EXPORT QgisApp : public QMainWindow, private Ui::MainWindow
     void activeLayerChanged( QgsMapLayer *layer );
 
   private:
+    //Flag to allow user to bypass badlayer checks.
+    bool skipBadLayers;
```

Comment:
```suggestion
    bool mSkipBadLayers;
```
Can you prefix this with `m` to align with the current coding standards.